src/NServiceBus.Transport.RabbitMQ/FodyWeavers.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Weavers>
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Weavers GenerateXsd="false">
```

Comment:
What does this do?

Reply:
New versions of Fody now want to generate an xml schema file on build, so that tells it not to.